.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,33 +1,36 @@
+<!-- Summarize the purpose or reasons for this PR, e.g. what problem it solves or what benefit it provides. -->
 
-Complete the following checkbox ONLY IF it is applicable to your PR. You can complete it later if it is not currently applicable:
--   [ ] I have verified all tests and linters pass against my changes, and/or I have included automated tests where applicable
 
-<hr>
+## This PR
+<!-- A bullet point list of one or more items describing the specific changes. -->
 
-**1. Because:**
-<!--
-If this PR closes an open issue, replace the XXXXX below with the issue number, e.g. Closes #2013. Or if the issue is in another TOP repo replace the #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX
 
-Otherwise, provide a clear and concise reason for your pull request, e.g. what problem it solves or what benefit it provides. If this PR is related to, but does not close, another issue or PR, you can also link it as above without the 'Closes' keyword, e.g. "Related to #2013".
- -->
-Closes #XXXXX
+## Issue
+<!--
+If this PR closes an open issue in this repo, replace the XXXXX below with the issue number, e.g. Closes #2013.
 
+If this PR closes an open issue in another TOP repo, replace the #XXXXX with the URL of the issue, e.g. Closes https://github.com/TheOdinProject/curriculum/issues/XXXXX
 
-**2. This PR:**
-<!--
-A bullet point list of one or more items outlining what was done in this PR to solve the problem(s) or implement the feature/enhancement.
- -->
+If this PR does not close, but is related to another issue or PR, you can link it as above without the 'Closes' keyword, e.g. 'Related to #2013'.
+-->
+Closes #XXXXX
 
+## Additional Information
+<!-- Any other information about this PR, such as a link to a Discord discussion. -->
 
-**3. Additional Information:**
-<!-- Any additional information about the PR, such as a link to a Discord discussion, etc. -->
 
+## Pull Request Requirements
+<!-- Replace the whitespace between the square brackets with an 'x', e.g. [x]. Or after you create the PR, they will become checkboxes that you can click on. -->
+-   [ ] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/theodinproject/blob/main/CONTRIBUTING.md)
+-   [ ] The title of this PR follows the `keyword: brief description of change` format, using one of the following keywords:
+  - `Feature` - adds new or amends existing user-facing behavior
+  - `Chore` - changes that have no user-facing value, refactors, dependency bumps, etc
+  - `Fix` - bug fixes
+-   [ ] The `Because` section summarizes the reason for this PR
+-   [ ] The `This PR` section has a bullet point list describing the changes in this PR
+-   [ ] I have verified all tests and linters pass after making these changes.
+-   [ ] If this PR addresses an open issue, it is linked in the `Issue` section
+-   [ ] If applicable, this PR includes new or updated automated tests
```

Comment:
For the app repo specifically, I think we should also add `-  [ ] I have squashed my commits`

Reply:
I'm not sure that I agree with adding this checkbox. I think it is an expectation of maintainers to do this, but not outside contributors. If we want outside contributors to do this, I think I would like to see an extended set of instructions on how to do this in the contributors guide. 

In addition, I think moving forward, we can use people's "Because" and "This PR" to be their commit message when we rebase & merge in their PR (which I believe has been recently enabled on this repo).

Reply:
@KevinMulhern Do you have a preference for adding a checkbox for "I have squashed my commits" (and putting more details in our contributing guide) or leaving it off for the time being and seeing how it goes with us being able to rebase & merge PRs?

Reply:
I'll defer to Kevin but I think this is an improvement either way, so not a blocker for me.

Definitely worth a bit more disucssion though - following that format is a good convention and it's more valuable the more consistent it is. But maybe it creates an extra barrier to entry for contribution. On balance I would be for forcing it as a requirement for all contributions, but you're right that we would probably want to flesh out the instructions.

[ This page in the wiki ](https://github.com/TheOdinProject/theodinproject/wiki/Squashing-Commits) covers most of it, but we could probably make that page more obvious.

Reply:
Yes, that page does cover it nicely, however if doesn't work that way for me anymore. I don't remember if it is because I've set up oh my zsh or made VS Code my default git editor. (the order is reversed, so I need to have the bottom one as pick) But yes, this does take a bit more discussion, so I'll merge in this PR for now and we'll come back to this in the future.
<img width="984" alt="Screen Shot 2022-10-21 at 9 14 51 PM" src="https://user-images.githubusercontent.com/33227096/197314841-3fd0848c-f231-4397-9c18-7ea5aba06b98.png">

Reply:
> Do you have a preference for adding a checkbox for "I have squashed my commits" (and putting more details in our contributing guide) or leaving it off for the time being and seeing how it goes with us being able to rebase & merge PRs?

I think it would be worth adding as a prompt/reminder to squash. But squash and rebase has been working very nicely lately with PR's from outside the maintainer team.